firebase-firestore/src/main/java/com/google/firebase/firestore/local/MemoryRemoteDocumentCache.java

```diff
@@ -53,6 +54,21 @@ public MaybeDocument get(DocumentKey key) {
     return docs.get(key);
   }
 
+  @Override
+  public Map<DocumentKey, MaybeDocument> getAll(Iterable<DocumentKey> keys) {
```

Comment:
One of the reasons I'm returning a `Map` and not an `ImmutableSortedMap` is because the latter doesn't have `keySet`/`values` methods, which are very useful.